### ISSUE TITLE
Parse error at `EXTENDED_ARG'

### ISSUE THREAD

**Issue by kurzweiler** [rocky/python-uncompyle6#75]
$> uncompyle6 XXX_2.cpython-35.pyc
```
# uncompyle6 version 2.9.9
# Python bytecode 3.5 (3350)
# Decompiled from: Python 3.5.1 (v3.5.1:37a07cee5969, Dec  6 2015, 15:54:25) [MSC v.1900 64 bit (AMD64)]
# Embedded file name: C:\Users\.....\XXX_2.py
# Compiled at: 2016-12-25 15:22:48
# Size of source mod 2**32: 615 bytes
Instruction context:

  12      12  LOAD_NAME                'XXX_1'
             15  LOAD_ATTR                'RContext'
             18  LOAD_CONST               ('ctx',)
             21  LOAD_CONST               '<code_object enterR>'
             24  LOAD_CONST               'XXX_2.enterR'
->	         27  EXTENDED_ARG          2  ''
             30  MAKE_FUNCTION_A_2_0        '0 positional, 0 keyword pair, 2 annotated'
             33  STORE_NAME               'enterR'

# file XXX_2.cpython-35.pyc
# --- This code section failed: ---

   9       0  LOAD_NAME                '__name__'
           3  STORE_NAME               '__module__'
           6  LOAD_CONST               'XXX_2'
           9  STORE_NAME               '__qualname__'

  12      12  LOAD_NAME                'XXX_1'
          15  LOAD_ATTR                'RContext'
          18  LOAD_CONST               ('ctx',)
          21  LOAD_CONST               '<code_object enterR>'
          24  LOAD_CONST               'XXX_2.enterR'
          27  EXTENDED_ARG          2  ''
          30  MAKE_FUNCTION_A_2_0        '0 positional, 0 keyword pair, 2 annotated'
          33  STORE_NAME               'enterR'

  16      36  LOAD_NAME                'XXX_1'
          39  LOAD_ATTR                'RContext'
          42  LOAD_CONST               ('ctx',)
          45  LOAD_CONST               '<code_object exitR>'
          48  LOAD_CONST               'XXX_2.exitR'
          51  EXTENDED_ARG          2  ''
          54  MAKE_FUNCTION_A_2_0        '0 positional, 0 keyword pair, 2 annotated'
          57  STORE_NAME               'exitR'

Parse error at or near `EXTENDED_ARG' instruction at offset 27
```

**Comment by rocky** [comment]
Looks like a small program. If you have the source code pass it along and that will simplify fixing. 

If you don't, or additionally, attach the bytecode file. (Maybe pycdc can decompile this and I can get the source).

**Comment by Zm908** [comment]
I have the same bug. A .pyc file is attached
[extarg.zip](https://github.com/rocky/python-uncompyle6/files/905649/extarg.zip)

**Comment by rocky** [comment]
@Zm906 While this is better, it is still not quite actionable. I need what source code should be produced for this bytecode. 

In your other bug reports I could fill that in from the filename that is embedded in the bytecode. Specifically they were from sqlalchemy. Here, the package name is not in the embedded, it is simply ` helpers/mailer/renderer/filters.py`.

**Comment by Zm908** [comment]
Unfortunately, I don't have the source code for that file. So, I decompiled the standard library and found some files with EXTENDED_ARG error. Source code, decompiler output, and the .pyc files are attached.
[extended_args.zip](https://github.com/rocky/python-uncompyle6/files/910229/extended_args.zip)

**Comment by rocky** [comment]
This was probably fixed in commit be9194c22381ded65c335c436bcc393bb6f8e390

Both of the two zip files decompile, so I'll assume @kurzweiler 's does too. If not feel free to open an new ticket with sufficient information, i.e. bytecode and corresponding source code. Narrowing the problem to just the function and/or essential statements in the function would be appreciated too.
